test/meson.build

```diff
@@ -19,8 +16,17 @@ nntrainer_testutil_dep = declare_dependency(
   include_directories: nntrainer_test_inc
 )
 
-nntrainer_test_deps += nntrainer_testutil_dep
-nntrainer_test_deps += gtest_dep
+nntrainer_test_deps = [
+  nntrainer_dep,
+  nntrainer_testutil_dep,
+  gtest_dep
+]
+
+nntrainer_test_main_deps = [
```

Comment:
please add a comment how this is different from `nntrainer_test_deps`.

Reply:
Added